dv-export/python/main.py

```diff
@@ -0,0 +1,59 @@
+import shutil
+
+from dotenv import dotenv_values
+from visier.connector import VisierSession, make_auth
+from visier.api import DVExportApiClient
+from data_store import *
+from dv_export import DVExport, DataVersions
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s [%(levelname)s] %(name)s: %(message)s',
+    filename='app.log',
+    filemode='w',
+)
+console_handler = logging.StreamHandler()
+console_handler.setLevel(logging.INFO)
+console_handler.setFormatter(logging.Formatter('%(asctime)s [%(levelname)s] %(name)s: %(message)s'))
+logger = logging.getLogger()
+logger.addHandler(console_handler)
+
+env_variables = dotenv_values()
```

Comment:
Consider splitting the `.env` file into two so you can more cleanly load creds as separate from other configuration.

Pattern [here](https://pypi.org/project/python-dotenv/):
```python
import os
from dotenv import dotenv_values

config = {
    **dotenv_values(".env.shared"),  # load shared development variables
    **dotenv_values(".env.secret"),  # load sensitive variables
    **os.environ,  # override loaded values with environment variables
}
```